strings.txt

```diff
@@ -16858,3 +16858,955 @@
+    cs = Garáž
+    it = Box
+    nb = Garasjebokser
+    zh-Hans = 停车房
+    th = โรงรถแบบกล่อง
+    ja = ガレージボックス
+    ar = صناديق جراج
+    sk = Hromadné garáže
+    es = Garaje
+    pl = Miejsca parkingowe w garażu
+    nl = Garageboxen
+
+  [placepage_parking_pay]
+    en = Pay
+    ru = Платная
+    fr = Payer
+    da = Betal
+    id = Berbayar
+    ko = 유료
+    sv = Kostnadsbelagd
+    tr = Ücretli
+    uk = Платне
+    vi = Trả phí
+    hu = Fizetős
+    de = Gebührenpflichtig
+    fi = Maksullinen
+    cs = Placené
+    it = A pagamento
+    nb = Betaling
+    zh-Hans = 付款
+    th = เสียเงิน
+    ja = 支払い
+    ro = Cu plată
+    ar = دفع
+    sk = Platené
+    es = De pago
+    pl = Płatny
+    nl = Betalen
+
+  [placepage_parking_free]
+    en = Free
+    ru = Бесплатная
+    fr = Gratuit
+    da = Gratis
+    id = Gratis
+    ko = 무료
+    sv = Gratis
+    tr = Ücretsiz
+    uk = Безкоштовне
+    vi = Miễn phí
+    hu = Díjmentes
+    de = Gebührenfrei
+    fi = Ilmainen
+    cs = Zdarma
+    it = Gratis
+    nb = Gratis
+    zh-Hans = 免费
+    th = ฟรี
+    ja = 無料
+    ro = Gratuit
+    ar = مجانًا
+    sk = Zdarma
+    es = Gratuito
+    pl = Darmowy
+    nl = Gratis
+
+  [placepage_parking_interval]
+    en = Interval Pay
+    ru = Переменно платная
+    fr = Période de paiement
+    da = Intervalbetaling
+    id = Biaya Rutin
+    ko = 주차요금 간격
+    sv = Delvis kostnadsbelagd
+    tr = Süre Ücretli
+    uk = Змінно платне
+    vi = Trả Phí Cách Quãng
+    hu = Idősávos fizetős
+    de = Teilweise gebührenpflichtig
+    fi = Aikamaksu
+    cs = Placeno za čas
+    it = Pagamento a tempo
+    nb = Intervallbetaling
+    zh-Hans = 间隔付款
+    th = จ่ายตามช่วงเวลา
+    ja = 支払間隔
+    ro = Interval de plată
+    ar = دفع عن فترة
+    sk = Platba v intervaloch
+    es = Pago por intervalos
+    pl = Płatny na godziny
+    nl = Intervalbetaling
+
+  [placepage_entrance_number]
+    comment = Entrance number in Place Page
+    en = No.
+    ru = №
+    fr = N°
+    da = Nr.
+    id = Nomor
+    ko = 번호
+    sv = Nr
+    tr = No.
+    uk = №
+    vi = Số
+    hu = Szám
+    de = Nr. der Einfahrt
+    fi = Nro.
+    cs = Č.
+    it = N.
+    nb = Nr.
+    zh-Hans = 编号
+    th = หมายเลข
+    ja = 番号
+    ro = Nr.
+    ar = رقم
+    sk = Číslo
+    es = N.º
+    pl = Nr
+    nl = Nr.
+
+  [placepage_entrance_type]
+    comment = Entrance type in Place Page
+    en = Entrance
+    ru = Подъезд
+    fr = Entrée
+    da = Indgang
+    id = Gerbang Masuk
+    ko = 입구
+    sv = Entré
+    tr = Giriş
+    uk = Під'їзд
+    vi = Lối vào
+    hu = Bejárat
+    de = Typ der Einfahrt
+    fi = Sisäänkäynti
+    cs = Vchod
+    it = Ingresso
+    nb = Vindfang
+    zh-Hans = 入口
+    th = ทางเข้า
+    ja = エントランス
+    ro = Intrare
+    ar = المدخل
+    sk = Vstup
+    es = Entrada
+    pl = Wejście
+    nl = Ingang
+
+  [placepage_flat]
+    comment = Flat number in place page
+    en = app.
+    ru = кв.
+    fr = app.
+    da = app.
```

Comment:
Похоже на то, да.
> On Apr 26, 2016, at 4:29 PM, Alex Zolotarev <notifications@github.com> wrote:
> 
> In strings.txt <https://github.com/mapsme/omim/pull/3005#discussion_r61084719>:
> 
> > +    zh-Hans = 入口
> > +    th = ทางเข้า
> > +    ja = エントランス
> > +    ro = Intrare
> > +    ar = المدخل
> > +    sk = Vstup
> > +    es = Entrada
> > +    pl = Wejście
> > +    nl = Ingang
> > +
> > +  [placepage_flat]
> > +    comment = Flat number in place page
> > +    en = app.
> > +    ru = кв.
> > +    fr = app.
> > +    da = app.
> На остальных языках лучше перепроверить, возможно была неправильная калька с английского app. и просто не поняли переводчики, к чему это.
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/mapsme/omim/pull/3005/files/c2b186b3a7f7730ba2f8b359702c3efcb561c295..454ca48abdc1a7a643a178fac869c249f100121f#r61084719>